.travis.yml

```diff
@@ -10,14 +11,10 @@ notifications:
   email: false
 git:
   depth: 99999999
-
 matrix:
   allow_failures:
   - julia: nightly
-
-after_success:
-  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
-
+  fast_finish: true
```

Comment:
What's the effect of this ? Just by curiosity

Reply:
https://blog.travis-ci.com/2013-11-27-fast-finishing-builds/

Reply:
Interesting, thanks !